jetty-slf4j-impl/src/main/java/org/eclipse/jetty/logging/JettyLoggingEvent.java

```diff
@@ -0,0 +1,107 @@
+//
+// ========================================================================
+// Copyright (c) 1995-2020 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under
+// the terms of the Eclipse Public License 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0
+//
+// This Source Code may also be made available under the following
+// Secondary Licenses when the conditions for such availability set
+// forth in the Eclipse Public License, v. 2.0 are satisfied:
+// the Apache License v2.0 which is available at
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+package org.eclipse.jetty.logging;
+
+import org.slf4j.Marker;
+import org.slf4j.event.Level;
+import org.slf4j.event.LoggingEvent;
+
+public class JettyLoggingEvent implements LoggingEvent
```

Comment:
Not needed, see above.

Reply:
Absolutely needed for our slf4j 2.0.0 (near) future

Reply:
Can you explain why it is needed for slf4j 2.0?  I'ver had a quick look at the API and most of it is unchanged and doesn't require events.  Even the LoggingEventBuilder API should only require a builder implementation and there is no compulsion for normal logging to use that path.